### DISCUSSION TITLE
Rpeat test shortcut

### DISCUSSION THREAD

**Discussion by MossabDiae** [monkeytypegame/monkeytype#2627] [Ideas] [answered]
## TLDR
> New test shortcut = quick tab mode
> Repeat test shortcut = ???

## Scenario
Let's say I'm using auto fail (for both accuracy and wpm) , when the test auto-fails I'm activating the option to start a new one with `quick tab mode` **BUT** sometimes I really want to repeat the test to get a better start or there are some words I want to master .

**Comment by userrand** [comment]
I am not sure I understand but I might be agreeing that it would be nice to have an option to repeat the test with a shortcut rather than having to type a command or click a button each time

**Comment by Miodec** [comment, marked as answer]
Thats a tradeoff you make when using quick tab mode. If you want to have the ability to quickly repeat a test ( tab tab enter ) you will need to disable quick tab.

**Reply by userrand** [reply]
Nice I did not know I could do that thank you (for readers, I think this is done by turning off quick restart).

That basically solves the problem if one completes the test but I am not sure what are all the options available if someone wishes to change the text (not restart it) in the middle of the test. This is possible with choosing enter for quick restart but then the restart button disappears. That button could have maybe been used with the tab key to restart the test during the test.

I personally do not need to change text while doing a test but maybe some people do.

At the moment it seems to me that the other option is to refresh the browser but then that might loose session information and require reclicking the text to start a test.
